src/renderer/utils/storageHelper.ts

```diff
@@ -21,7 +21,7 @@ export interface StorageHelperOptions<T> {
   autoInit?: boolean; // start preloading data immediately, default: true
   observable?: CreateObservableOptions;
   storage: StorageAdapter<T>;
-  defaultValue?: T;
+  defaultValue: T;
```

Comment:
Why it should be required field? Why `defaultValue` cannot be `undefined`?

Reply:
Because `undefined` is not a valid `number`.

Reply:
Typescript does not support conditional optionality. But I have make a `StorageHelper` for `number`'s then we should have a defined default value.

Reply:
Didn't get what you mean about conditional optionality..

When you store a number you should define a `number` as `defaultValue` in place of use if it's cannot be `undefined` initially for your logic.

Reply:
Summoning @jakolehm to the thread. Do you have some thoughts on this?